client/src/components/createSubmissionComponents/SubmissionUpload.js

```diff
@@ -45,14 +59,15 @@ const SubmissionUpload = () => {
           },
         }
       );
-      const submission = res.data;
-      console.log(submission);
+      console.log(res.data);
```

Comment:
Are you doing anything with res.data? If not, you don't need to store the response in a variable

Reply:
Yes I need to get comfortable with not needing to see console.logs once I know a feature works. Thanks.